src/coreclr/vm/amd64/RedirectedHandledJITCase.asm

```diff
@@ -185,6 +185,7 @@ NESTED_ENTRY RedirectForThrowControl2, _TEXT
 
         save_reg_postrsp    rcx, REDIRECT_FOR_THROW_CONTROL_FRAME_SIZE + 8h     ; FaultingExceptionFrame
         save_reg_postrsp    rdx, REDIRECT_FOR_THROW_CONTROL_FRAME_SIZE + 10h    ; Original RSP
+        save_reg_postrsp    r8, REDIRECT_FOR_THROW_CONTROL_FRAME_SIZE + 18h     ; SSP
```

Comment:
I've changed the code to pass the SSP to the target function instead of storing it in the `FaultingExceptionFrame` so that the `FaultingExceptionFrame::Init` can zero it by default that's needed in all cases other than the funceval redirection.